datafusion/sql/tests/sql_integration.rs

```diff
@@ -4278,6 +4294,23 @@ fn test_table_alias() {
         \n          TableScan: person";
     quick_test(sql, expected);
 
+    let sql = "select * from (\
```

Comment:
I think we getting closer, please move those tests to `select.rs` you may want to use `EXPLAIN` keyword to get the plan

Reply:
Done.